edgelet/edgelet-docker/Cargo.toml

```
@@ -20,7 +20,7 @@ serde_json = "1"
 serial_test = "0.8.0"
 sysinfo = "0.25"
 thiserror = "1"
-tokio = "1"
+tokio = { version = "1", features = ["parking_lot"] }
```

```suggestion
tokio = { version = "1", features = ["parking_lot", "sync"] }
```
`parking_lot` does not expose `sync`. The development dependencies should include `macros` and `rt` since the tests use `#[tokio::test]`.